test/mpi/pt2pt/pingping.c

```diff
@@ -160,15 +160,15 @@ int main(int argc, char *argv[])
                     }
 
                     MTestCopyContent(recvbuf, recvbuf_h, recv_obj.DTP_bufsize, recvmem);
-                    err = DTP_obj_buf_check(recv_obj, recvbuf_h, 0, 1, count[0]);
+                    err = DTP_obj_buf_check(recv_obj, recvbuf_h, 0, 1, count[1]);
```

Comment:
Re-reading the test, apparently we only send/recv a single count in all pt2pt dtp tests. This is because the test iterates over multiple comms thus we can't initialize different dtp pool for different processes. I wonder how above check didn't break the test after the change.